### ISSUE TITLE
Provide Git binary

### ISSUE THREAD

**Issue by Beanow** [klakegg/docker-hugo#8]
The Git binary is used for `enableGitInfo = true` in the hugo config.
Themes commonly use this for "last modified" metadata and linking to commits on GitHub.

Missing this binary results in:
```
$ docker run --rm -tiv `pwd`:/src klakegg/hugo:0.55.6-ext
Building sites … ERROR 2019/08/06 19:09:33 Failed to read Git log: Git executable not found in $PATH
```

**Comment by klakegg** [comment]
Thank you very much for your feedback.

I've added git to the ext-builds, and will be available in images starting with 0.56.0.

PS. You may find the Alpine edition (`klakegg/hugo:0.xx.x-ext-alpine`) a bit lighter unless you need the Debian edition.

**Comment by Beanow** [comment]
Thank you for the quick update :smile:
I've tested with the Docsy theme again which includes scss, git and postcss dependencies.
Ignoring cache for good measure as that threw me off when testing with dependencies (locally, not from a container) before.

```
$ docker run --rm -tiv `pwd`:/src -v `pwd`/out:/target klakegg/hugo:0.56.0-ext-alpine --cleanDestinationDir --ignoreCache --verbose
Unable to find image 'klakegg/hugo:0.56.0-ext-alpine' locally
0.56.0-ext-alpine: Pulling from klakegg/hugo
bdf0201b3a05: Pull complete 
3927f1222b65: Pull complete 
b8a5b03dd959: Pull complete 
a9c18a16306f: Pull complete 
b9cf1df53f36: Pull complete 
fb7c9e5610ab: Pull complete 
Digest: sha256:e60049af58b1b6ef4c2e647359ac4dc8796304b2b2da27ccd8d9d4ff7588c39a
Status: Downloaded newer image for klakegg/hugo:0.56.0-ext-alpine
INFO 2019/08/07 09:56:49 Using config file: 
Building sites … INFO 2019/08/07 09:56:49 removing all files from destination that don't exist in static dirs
INFO 2019/08/07 09:56:49 syncing static files to /target/
INFO 2019/08/07 09:56:49 found taxonomies: map[string]string{"category":"categories", "tag":"tags"}
INFO 2019/08/07 09:56:49 postcss: use config file /src/themes/docsy/postcss.config.js

  Replace Autoprefixer browsers option to Browserslist config.
  Use browserslist key in package.json or .browserslistrc file.

  Using browsers option cause some error. Browserslist config 
  can be used for Babel, Autoprefixer, postcss-normalize and other tools.

  If you really need to use option, rename it to overrideBrowserslist.

  Learn more at:
  https://github.com/browserslist/browserslist#readme
  https://twitter.com/browserslist



                   | EN  
+------------------+----+
  Pages            | 54  
  Paginator pages  |  0  
  Non-page files   |  0  
  Static files     | 36  
  Processed images |  0  
  Aliases          |  3  
  Sitemaps         |  1  
  Cleaned          |  0  

Total in 1240 ms
```

The git data is rendered correctly :+1:
